q2_quality_control/_blast.py

```diff
@@ -72,7 +87,17 @@ def _extract_hits(blast_output):
                 and line != ""
                 # if vsearch fails to find assignment, it reports '*' as the
                 # accession ID, so we will not count those IDs as hits.
-                and line.split('\t')[1] != '*']
+                and line.split('\t')[1] != '*'
```

Comment:
`line` is being split four times here, which is adding some unnecessary computation. I recommend making this a `for` loop rather than a list comprehension (which is clearer anyway, where there are a lot of conditions), and splitting only one time (e.g., `fields = line.split('\t')`).

Reply:
👍